app/api/error_handler.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorHandler < Grape::Middleware::Base
+  def call!(env)
+    @env = env
+    begin
+      @app.call(@env)
+    rescue StandardError => e
+      throw :error, message: e.message || options[:default_message], status: 500
```

Comment:
could you please read about errors and why have to avoid raising 500 errors on our own